scripts/generate-database-data.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+"""
+Fill a PostgresSQL database with enough randomly generated data to run the DMp. Currently only creates a single buyer
```

Comment:
Is this description true, given that you're using the data API?

Reply:
I've updated it. This description is from when we thought the script would need to interact directly with the database